packaging/macOS/parsec.spec

```diff
@@ -0,0 +1,61 @@
+# -*- mode: python ; coding: utf-8 -*-
```

Comment:
1 - License header is missing
2 - `# -*-` encoding line is not needed since python3 so it can be removed
3 - we should use black on this file just to clean it a bit (you can rename it as `parsec.py`, run black on it, then rename it back to `parsec.spec`)